quiz.py

```diff
@@ -0,0 +1,148 @@
+import random
+from abc import ABCMeta, abstractmethod
+from sqlalchemy.sql.expression import exists
+from sqlalchemy.sql.functions import func
+from models.painter import Painter
+from models.painting import Painting
+from models.saint import Saint
+
+
+class MultipleChoiceQuestion:
+    def __init__(self, element, correct_option, option_list):
+        self._element = element
```

Comment:
1.) I'll give proper names
2.) It is supposed to be private. Where is it outside?
3.) Long story short, its better for checking with index.